index.html

```diff
@@ -72,5 +88,183 @@
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+	</div>
+</section>
+<br> <br> <br>
+<section class="head">
+	<!-- Heading Section  -->
+	<div class="flex-about">
+	  <div class="about-me">
+	  	<div class="head-title"> 
+	    	About Me 
+	    </div>
+	  	<div class="head-body">
+	  		Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+	  	</div>
+	  	<div class="see-3">
+	  		Get My Resume
+	  	</div>
```

Comment:
- [ ] Kindly be noticed that the button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).